CHANGELOG.md

```diff
@@ -1,4 +1,14 @@
 # CHANGELOG
+## 3.5.0
+#### New Functionality
+
+- Can now use BM25 and vector scoring method. (MarkLogic Server version 12.0.0-EA or higher needed).
```

Comment:
Since they are distinct features, how about two bullets:
Can now use BM25 for search scoring.
Can now use vector functionality including Cosine Similarity and others.

Reply:
sorry, just noticed Rob's comment. Closing this one.